src/Akeneo/Tool/Bundle/ApiBundle/EventSubscriber/CheckRequestContentSizeSubscriber.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Tool\Bundle\ApiBundle\EventSubscriber;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * This subscriber will watch the API auth route and throw a 413 if the request content size is too large.
+ * This is to mitigate DDoS attacks on this unrestricted endpoint.
+ */
+final class CheckRequestContentSizeSubscriber implements EventSubscriberInterface
```

Comment:
Ah. My previous comment is not good because this class is specialized thanks to the API_AUTH_ROUTE constant.

```suggestion
final class CheckApiRequestContentSizeSubscriber implements EventSubscriberInterface
```
In this case, could we not use a specialized naming?

Reply:
Or make it more configurable...